CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## master (unreleased)
 
+- Add user-defined exception handling ([@skryukov][])
+
+By default, Logidze raises an exception which causes the entire transaction to fail.
+To change this behavior, it's now possible to override `logidze_capture_exception(error_data jsonb)` function. 
+
+- [Fixes [#69](https://github.com/palkan/logidze/issues/69)] Fallback on NUMERIC_VALUE_OUT_OF_RANGE exception ([@skryukov][])
```

Comment:
There's a command to run to migrate the function, right?

```
bundle exec rails generate logidze:install --update
```

(assuming the pending migration warning is off)

https://github.com/palkan/logidze/blob/ad858c99a130ce4efa610e642d352d0cb8d03aaf/README.md#upgrading

Reply:
Yes, that's it. You should also run `bundle exec rails db:migrate` then =)